src/app/users/users-table.component.html

```diff
@@ -88,45 +88,15 @@
         <mat-chip-list>
           <mat-chip *ngIf="element.doc.roles.length > 0 || element.doc.isUserAdmin" i18n>Learner</mat-chip>
           <ng-container *ngFor="let role of element.doc.roles; index as i">
-            <mat-chip *ngIf="role !== 'learner'" [removable]="!element.doc.isUserAdmin" (removed)="removeRole(element.doc,i)">
+            <mat-chip *ngIf="role !== 'learner'">
               <planet-role [role]="role"></planet-role>
-              <mat-icon matChipRemove *ngIf="isUserAdmin && !element.doc.isUserAdmin">cancel</mat-icon>
             </mat-chip>
           </ng-container>
           <mat-chip *ngIf="element.doc.isUserAdmin && element.doc.roles.length === 0">Admin</mat-chip>
         </mat-chip-list>
         <span *ngIf="!element.doc.isUserAdmin && element.doc.roles.length === 0" i18n>Inactive</span>
       </mat-cell>
     </ng-container>
-    <ng-container matColumnDef="action">
```

Comment:
column should still be there but should be controlled by displayedcolumns property from component